### PR TITLE
SMTP

### PR DESCRIPTION
**Pull request by SolianHelix**

### CODE REVIEW

File: database/seeds/InventoriesTableSeeder.php [hunk 1]
```diff
@@ -13,17 +13,17 @@ class InventoriesTableSeeder extends Seeder
...
-            ['quantity' => '6', 'low' => 1, 'high' => 0],
-            ['quantity' => '7', 'low' => 1, 'high' => 0],
-            ['quantity' => '8', 'low' => 1, 'high' => 0],
-            ['quantity' => '5', 'low' => 1, 'high' => 0],
-            ['quantity' => '4', 'low' => 1, 'high' => 0],
-            ['quantity' => '3', 'low' => 1, 'high' => 0],
-            ['quantity' => '5', 'low' => 1, 'high' => 0],
-            ['quantity' => '77', 'low' => 0, 'high' => 1],
-            ['quantity' => '55', 'low' => 0, 'high' => 1],
+            ['id' => '1', 'ingredient_id' => '1', 'quantity' => '1', 'low' => 1, 'high' => 0],
```
Comment (Tyler-Ouellette) [inline database/seeds/InventoriesTableSeeder.php]: i know this was an issue with your table before merge but you can remove the ingred_id now i believe before merging

File: public/web.config [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!--
```
Comment (Tyler-Ouellette) [inline public/web.config]: what is this for? email?

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<p align="center"><img src="https://laravel.com/assets/img/components/logo-laravel.svg"></p>
```
Comment (Tyler-Ouellette) [inline readme.md]: ?

File: resources/js/components/ExampleComponent.vue [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<template>
```
Comment (Tyler-Ouellette) [inline resources/js/components/ExampleComponent.vue]: can delete this

File: database/seeds/InventoriesTableSeeder.php (hunk 1 above)
Comment (SolianHelix) [reply database/seeds/InventoriesTableSeeder.php]: Yeah this was just a work around for the time being so I could get the seeder to work properly, forgot to change it to use a factory like the master :X

File: resources/js/components/ExampleComponent.vue (hunk 4 above)
Comment (Tyler-Ouellette) [reply resources/js/components/ExampleComponent.vue]: like the whole  file

File: database/seeds/InventoriesTableSeeder.php (hunk 1 above)
Comment (Tyler-Ouellette) [reply database/seeds/InventoriesTableSeeder.php]: maybe do a git merge master into your branch

File: public/web.config (hunk 2 above)
Comment (SolianHelix) [reply public/web.config]: Looks like it rewrites links within emails. This was something that was auto-generated with make:mail I believe

Comment (Tyler-Ouellette) [reply public/web.config]: cool just wanted to make sure it wasn't like a windows vs mac thing.

**Pull request merged**